charts/karpenter/templates/deployment.yaml

```diff
@@ -99,6 +102,32 @@ spec:
                   containerName: controller
                   divisor: "0"
                   resource: limits.memory
+            - name: FEATURE_GATES
```

Comment:
What's the behavior if we don't specify any feature gates?

Reply:
It is an option with default, which will be used if the value is not specified. In v1.0.x [the default is](https://github.com/kubernetes-sigs/karpenter/blob/7b24b44084d10c028345bb0b2acbd6633af822c6/pkg/operator/options/options.go#L102) "SpotToSpotConsolidation=false", but it is likely to change in v1.1.x. In our provider we would want to keep it false until properly supported and tested.